evoxbench/database/mosegnas/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Please create your test here :P
or delete this file.

Reply:
I think it is better to simply put the empty test file here, because this is more convenient for others who need it to write tests. Also, all the previous models do have an empty test file.

Reply:
Like this one:
https://github.com/EMI-Group/evoxbench/blob/main/evoxbench/database/darts/tests.py

Reply:
Like this one:
https://github.com/EMI-Group/evoxbench/blob/8b18b1ec256dce179f470e9fc0d6c6422fad21b3/evoxbench/database/darts/tests.py

Reply:
ok that makes sense